SeqVision/SeqVision.py

```diff
@@ -0,0 +1,143 @@
+from typing import List, Union, Callable, Dict, Tuple
+import os
+from modules.fastq_tool_module import (
+    calculate_gc_content,
+    filter_reads_by_gc,
+    filter_length_bounds,
+    filter_quality_threshold,
+)
+from modules.dna_rna_tools_module import (
+    is_rna,
+    is_dna,
+    complement,
+    reverse,
+    transcribe,
+    reverse_complement,
+    which_palindrome,
+)
+
+
+def filter_fastq(
+    input_fastq: str,
+    output_fastq: str,
+    gc_bounds: Union[Tuple[int, int], int, float] = (0, 100),
```

Comment:
Так как в Tuple тоже могут передать или int, или float, можно было бы немного дополнить аннотацию
Но в целом, во всём коде много внимания уделено аннотации типов, всё очень круто и уважаемо!
```suggestion
    gc_bounds: Union[
            Tuple[Union[int, float], Union[int, float]],
            int,
            float
        ] = (0, 100),
```